setuptools_rust/setuptools_ext.py

```diff
@@ -19,9 +19,12 @@
 from .extension import Binding, RustBin, RustExtension, Strip
 
 try:
-    from wheel.bdist_wheel import bdist_wheel
+    from setuptools.command.bdist_wheel import bdist_wheel
 except ImportError:
-    bdist_wheel = None
+    try:  # old version of setuptools
+        from wheel.bdist_wheel import bdist_wheel
+    except ImportError:
+        bdist_wheel = None
```

Comment:
This is a bit ugly (too nested), but similar to other code in the same file...
Alternatively we could also shove this conditional import into a separated file to hide the mess.